src/check/parse.zig

```diff
@@ -95,17 +90,41 @@ fn tokenizeReport(allocator: std.mem.Allocator, source: []const u8, msgs: []cons
     }
 }
 
+// TODO move this somewhere better, for now it's here to keep it simple.
+fn testSExprHelper(source: []const u8, expected: []const u8) !void {
+    var env = base.ModuleEnv.init(testing.allocator);
```

Comment:
This is also missing deinit.
```suggestion
    var env = base.ModuleEnv.init(testing.allocator);
    defer env.deinit()
```